src/main/java/cs361/battleships/models/Result.java

```diff
@@ -7,32 +7,43 @@
 	@JsonProperty private AtackStatus result;
 	@JsonProperty private Square location;
 	@JsonProperty private Ship ship;
+	@JsonProperty private boolean submerged;
 	@SuppressWarnings("unused")
+
 	public Result() {
 	}
 
 	public Result(Square location) {
-		result = AtackStatus.MISS;
+		this.result = AtackStatus.MISS;
 		this.location = location;
+		this.submerged = false;
 	}
 
 	public AtackStatus getResult() {
-		return result;
+		return this.result;
 	}
 
 	public void setResult(AtackStatus result) {
 		this.result = result;
 	}
 
 	public Ship getShip() {
-		return ship;
+		return this.ship;
 	}
 
 	public void setShip(Ship ship) {
 		this.ship = ship;
 	}
 
 	public Square getLocation() {
-		return location;
+		return this.location;
+	}
+
+	public boolean getSubmerged() {
+		return this.submerged;
+	}
+
+	public void setSubmerged() {
```

Comment:
This should be like 
setSubmerged(boolean val) {
    this.submerged = val;
}

Reply:
Great, we can make those changes!